gst/tensor_ros_src/tensor_ros_src.cc

```diff
@@ -0,0 +1,464 @@
+/*
+ * GStreamer
+ * Copyright (C) 2005 Thomas Vander Stichele <thomas@apestaart.org>
+ * Copyright (C) 2005 Ronald S. Bultje <rbultje@ronald.bitfreak.net>
+ * Copyright (C) 2019 Samsung Electronics Co., Ltd.
+ * 
+ * This library is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU Library General Public
+ * License as published by the Free Software Foundation; either
+ * version 2 of the License, or (at your option) any later version.
+ *
+ * This library is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ * Library General Public License for more details.
+ */
+
+/**
+ * SECTION:element-tensor_ros_src
+ *
+ * Source element to publish tensor stream from deginated Ros topic
+ *
+ * @file    tensor_ros_src.cc
+ * @date    03/06/2019
+ * @brief   GStreamer plugin to subscribe a Ros topic and convert them into tensor stream
+ * @see     https://github.com/nnsuite/nnstreamer-ros
+ * @author  Sangjung Woo <sangjung.woo@samsung.com>
+ * @bug     No known bugs except for NYI items
+ */
+
+#ifdef HAVE_CONFIG_H
+#  include <config.h>
+#endif
+
+#include <tensor_filter_custom.h>
```

Comment:
This is not a custom filter. You should not need this.

Reply:
Right.  I will.